lib/frameworks/cucumber.js

```diff
@@ -147,9 +159,13 @@ exports.run = function(runner, specs) {
     return q.promise(function(resolve, reject) {
       var cucumberConf = Cucumber.Cli.Configuration(execOptions);
       var runtime = Cucumber.Runtime(cucumberConf);
-      var formatter = cucumberConf.getFormatter();
-      addResultListener(formatter);
-      runtime.attachListener(formatter);
+      var formatters = cucumberConf.getFormatter ?
+        [cucumberConf.getFormatter()] :
+        cucumberConf.getFormatters();
+
+      addResultListener(formatters[0]);
```

Comment:
Yes, protractor just needs 1 of the formatters to hook into in order to figure out which scenarios passed/failed. All of the formatters have the same api so it doesn't matter which one we use.